From c4e947da066ee6ccb527e71fb861eccd14e7e4bb Mon Sep 17 00:00:00 2001
From: YunTaeKyun <98891717+YunTaeKyun93@users.noreply.github.com>
Date: Tue, 19 Mar 2024 11:53:54 +0900
Subject: [PATCH] Update tigers.yaml

---
 tigers.yaml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tigers.yaml b/tigers.yaml
index 0dd18ff..a1d3d65 100644
--- a/tigers.yaml
+++ b/tigers.yaml
@@ -10,5 +10,5 @@ members:
 
 - David
 fadsf
-fetch : This
+fetch : This!!!!
 
-- 
2.41.0

